File: Beecrowd/Beginner/1101 - 1190/1115 - Quadrant/Python/1115 - Quadrant.py
while True:
    x, y = map(int, input().split())

    if x == 0 or y == 0:
        break

    if x > 0 and y > 0:
        print("primeiro")
    elif x < 0 and y > 0:
        print("segundo")
    elif x < 0 and y < 0:
        print("terceiro")
    else:
        print("quarto")

File: Beecrowd/1 - Beginner/1101 - 1190/1115 - Quadrant/C++/1115 - Quadrant.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int x, y;

    while (true) {
        cin >> x >> y;

        if (x == 0 || y == 0) break;

        if (x > 0 && y > 0) cout << "primeiro" << endl;
        else if (x < 0 && y > 0) cout << "segundo" << endl;
        else if (x < 0 && y < 0) cout << "terceiro" << endl;
        else cout << "quarto" << endl;
    }
    
    return 0;
}